joplin/settings.py

```diff
@@ -182,7 +182,8 @@
 
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
-
+HEROKU_KEY = os.getenv('HEROKU_KEY')
+HEROKU_JANIS_APP_NAME = os.getenv('HEROKU_JANIS_APP_NAME')
```

Comment:
So I think I understand how we would get janis-staging or austindotgov heroku apps to rebuild upon a page publish from local based on your instructions.

But in order for this to work not just from local, but from a (un)publish event in `joplin-staging` or `joplin` we already have the env vars we need set, right?

Reply:
I can't login to heroku, but I think I added the vars as stubs. To get it fully setup, I think you'll need to create a new user and add a non-expiring token for them. The [auth docs](https://devcenter.heroku.com/articles/platform-api-quickstart#authentication) say

> You can create a non-expiring token by running heroku authorizations:create

Then you'll need to add that token to the config vars section of the app(s).